edge/pkg/edgehub/certificate/certmanager.go

```diff
@@ -138,7 +138,7 @@ func (cm *CertManager) applyCerts() error {
 	}
 
 	// get the edge.crt
-	caPem := pem.EncodeToMemory(&pem.Block{Bytes: cacert, Type: "CERTIFICATE"})
+	caPem := pem.EncodeToMemory(&pem.Block{Bytes: cacert, Type: certutil.CertificateBlockType})
```

Comment:
For the `CertificateBlockType` in `edge/pkg/edgehub/common/certutil/certfile.go`, let's also use the certutil from `client-go`.